### PR TITLE
Fix failing macos workflow

### LINKED ISSUE #68: Unit tests currently failing on MacOS
See <https://github.com/aiproteomics/aiproteomics/actions/runs/9223343210/job/25376350361>
Strange as no change was made to the tests - perhaps something deprecated or changed in the runner? Tests passing on ubuntu runner.

### PR DESCRIPTION
**Pull request by JaroCamphuijsen**
Macos build fails due to tensorflow import issues. 

This issue describes the problem from the side of poetry: https://github.com/python-poetry/poetry/issues/8271

fixes #68

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,8 +16,26 @@ pydot = "^1.4.2"
...
 lxml = "^4.9.2"
 seaborn = "^0.12.2"
+
+# Issue between poetry and tensorflow metadata since >=2.11
+# This is a temporary workaround
+# related to https://github.com/python-poetry/poetry/issues/8271
+# Inspired from https://github.com/tensorflow/tensorflow/blob/adb39b04e9cb116df4659a7e2de9eea27e62f25c/tensorflow/tools/pip_package/setup.py#L148-L162
+tensorflow = {version = "2.15.0" }
+tensorflow-macos = { version = "2.15.0", platform = "darwin" }
+tensorflow-cpu = [
```
Comment (JaroCamphuijsen) [inline pyproject.toml:26]: This line will throw an error in prospector linting. Somehow it doesn't agree with it being a list.

**Comment by JaroCamphuijsen** [comment]
Ready for review @raar1

File: pyproject.toml (hunk 1 above)
Comment (raar1) [reply pyproject.toml:27]: Strange indeed.

**Review by raar1 — APPROVED** [review]
Thanks a lot @JaroCamphuijsen

**Pull request merged**